src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -148,5 +184,78 @@ public Contig apply(String contigString) {
 
     return contigs;
   }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
```

Comment:
We're using 4 different words to mean the same thing: contig, shard, region, and interval. Let's try to focus on just one. Unless we want to rename this class, I suggest that our best candidate is "contig". We'll use that to mean "a contiguous region of the genome". 

So this function can be described as (i) loading the specified variants via the API, (ii) casting those variants to contigs - i.e. only keeping the information about where the variant is.

I'm a bit confused about why this should be here. Loading variants via the API has nothing to do with contigs so it doesn't feel like it belongs here. The second part sounds like it should be a method on the variant object: variant.getContig(). Surely that either already exists, or we should add it because knowing where a variant is seems fundamental.

Given this, I don't see why either step belongs here. Dataflow is good at combining DoFns, so it's already easy enough to write variantIDs.apply(new LoadVariantFn()).apply(new CastToContigFn()). No?